src/DssSpell.sol

```diff
@@ -99,7 +189,7 @@ contract DssSpell {
     // This should be modified weekly to provide a summary of the actions
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/7545776ce23113335369331dff39d71d9e767a19/governance/votes/Executive%20vote%20-%20January%2018%2C%202021.md -q -O - 2>/dev/null)"
     string constant public description =
-        "2021-01-18 MakerDAO Executive Spell | Hash: 0x71eb4d0a0f678bc2d706033c2ad238e637fb7665521040a31a193ad27d89183c";
+        "2021-01-22 MakerDAO Executive Spell | Hash: 0x71eb4d0a0f678bc2d706033c2ad238e637fb7665521040a31a193ad27d89183c";
```

Comment:
You should zero the hash and comment link out above so that it doesn't get overlooked later.